<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>202</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>50868162f5015a20ea4511430fdf0285.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fizyczny kalkulator</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>a=F/m</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>460</width>
           <height>501</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="aFmPic">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Sila</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QComboBox" name="aFmFType">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="currentIndex">
                   <number>5</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>[n</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[µ</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[m</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[c</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[d</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[da</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[h</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[k</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[M</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[G</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="aFmFInp">
                  <property name="inputMask">
                   <string notr="true"/>
                  </property>
                  <property name="frame">
                   <bool>true</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>N]</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Masa</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QComboBox" name="aFmMType">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="currentIndex">
                   <number>8</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>[n</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[µ</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[m</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[c</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[d</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[da</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[h</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[k</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[M</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[G</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="aFmMInp">
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>g]</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Przyspieszenie</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QComboBox" name="aFmSType">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="currentIndex">
                   <number>5</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>[nm</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[µm</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[mm</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[cm</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[dm</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[ m</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[dam</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[hm</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[km</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[Mm</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>[Gm</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="aFmSInp">
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="placeholderText">
                   <string>s²]</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="aFmBtn">
            <property name="text">
             <string>Obliczyć</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="aFmOutp"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
